centinel/models.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, **kwargs):
         # type
 
         allowed_keys = {"username": "string", "is_vpn": bool,
+                        "registered_date": datetime,
```

Comment:
Two questions about this:
1) Is this storing the actual datetime or just the method? I thought you needed to call something like datetime.utcnow() to get the date.
2) Do we want to perturb the data by adding noise to it? For every piece of data we collect, we need to consider a) how much accuracy we need vs b) the risk that users take on for each additional increment of accuracy